index.js

```diff
@@ -182,62 +182,79 @@ const TinderCard = React.forwardRef(({ flickOnSwipe = true, children, onSwipe, o
     swipeAlreadyReleased.current = false
   }, [swipeAlreadyReleased])
 
-  React.useLayoutEffect(() => {
-    let offset = { x: null, y: null }
-    let speed = { x: 0, y: 0 }
-    let lastLocation = { x: 0, y: 0, time: new Date().getTime() }
-    let mouseIsClicked = false
+  const [offset, setOffset] = React.useState({ x: null, y: null })
```

Comment:
@3DJakob needed to move these to state, since they would get 'reset' every time useLayoutEffect was called. So if it got called on an element you were mid-way through dragging then the element would jump to the wrong position.

Reply:
(I'm open to better options that don't require re-calling `useLayoutEffect` many times)